tests/lib/config/config-ops.js

```diff
@@ -481,4 +481,151 @@ describe("ConfigOps", function() {
 
     });
 
+    describe("normalize()", function() {
+        it("should convert error rule setting to 2 when rule has just a severity", function() {
+            var config = {
+                rules: {
+                    foo: "error",
+                    bar: "error"
+                }
+            };
+
+            ConfigOps.normalize(config);
+
+            assert.deepEqual(config, {
+                rules: {
+                    foo: 2,
+                    bar: 2
+                }
+            });
+        });
+
+        it("should convert error rule setting to 2 when rule has array with severity", function() {
+            var config = {
+                rules: {
+                    foo: ["error", "something"],
+                    bar: "error"
```

Comment:
Might be good to test more complex configs. Do all the various combinations works.